x-pack/plugins/infra/server/lib/alerting/metric_threshold/register_metric_threshold_alert_type.ts

```diff
@@ -55,6 +55,29 @@ export function registerMetricThresholdAlertType(libs: InfraBackendLibs) {
     }
   );
 
+  const valueActionVariableDescription = i18n.translate(
+    'xpack.infra.metrics.alerting.threshold.alerting.valueActionVariableDescription',
+    {
+      defaultMessage:
+        'The value of the metric in the specified condition. Usage: (ctx.value.condition0).',
```

Comment:
Maybe `Usage: (ctx.value.condition0, ctx.value.condition1, etc...)`?

Reply:
works for me.